root/io/filemerger/execTestMultiMerge.C

```diff
@@ -67,26 +67,27 @@ int testSimpleFile(const char *filename, Long64_t entries, Int_t compSetting, Lo
    return 0;
 }
 
-
 int execTestMultiMerge()
 {
    Int_t result = 0;
-   result += testMergedFile("mfile1-4.root",1,4938, kIs32bits ? 3 : 0);
+   int hsimpleFTolerance = 6; // 5 for non fst builds
    result += testMergedFile("mzfile1-4.root",206,4992, kIs32bits ? 2 : 0);
-
+   result += testMergedFile("mlz4file1-4.root",406,5013, kIs32bits ? 2 : 0);
+   result += testMergedFile("mzlibfile1-4.root",106,4917, kIs32bits ? 2 : 0);
    result += testSimpleFile("hsimple.root",25000,1,414415, kIs32bits ? 10 : 8);
-   result += testSimpleFile("hsimple9.root",25000,9,432029,3);
-   result += testSimpleFile("hsimple9x2.root",2*25000,9,851123,9);
+   result += testSimpleFile("hsimple9.root",25000,109,432030,3);
```

Comment:
For clarity, can you add hsimple101, hsimple109 and hsimple109x2 and 'revert' hsimple9 and hsimple9x2 to use the default?  Thanks.